tests/langchain/test_langchain_databricks_dependency_extraction.py

```diff
@@ -123,7 +123,10 @@ def test_parsing_dependency_from_databricks_retriever(monkeypatch: pytest.Monkey
     from langchain.vectorstores import DatabricksVectorSearch
```

Comment:
We have two tests with exactly same name `test_parsing_dependency_from_databricks_retriever`. Pytest only runs one of them. Apparently the test contents are exactly same aside from the minor import path difference. Can you remove the duplication?